src/services/infra/StatsService.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable import/prefer-default-export */
+import StatsDClient, { StatsD } from "hot-shots"
+import _ from "lodash"
+import { ModelStatic } from "sequelize"
+
+import { config } from "@config/config"
+
+import {
+  GH_MAX_REPO_COUNT,
+  GITHUB_ORG_REPOS_ENDPOINT,
+  ISOMERPAGES_REPO_PAGE_COUNT,
+} from "@root/constants"
+import { AccessToken, Site, User } from "@root/database/models"
+
+import { genericGitHubAxiosInstance } from "../api/AxiosInstance"
+
+export class StatsService {
+  private readonly statsD: StatsD
+
+  private readonly usersRepo: ModelStatic<User>
+
+  private readonly accessTokenRepo: ModelStatic<AccessToken>
+
+  private readonly sitesRepo: ModelStatic<Site>
+
+  constructor(
+    statsDClient: StatsD,
+    usersRepo: ModelStatic<User>,
+    accessTokenRepo: ModelStatic<AccessToken>,
+    sitesRepo: ModelStatic<Site>
+  ) {
+    this.statsD = statsDClient
+    this.usersRepo = usersRepo
+    this.accessTokenRepo = accessTokenRepo
+    this.sitesRepo = sitesRepo
+  }
+
+  submitV1Count = (path: string) => {
+    this.statsD.increment("versions.v1", {
+      version: "v1",
+      path,
+    })
+  }
+
+  countDbUsers = async () => {
+    // NOTE: Track only active users.
```

Comment:
Similar to another comment, why are we tracking this per user login when a user logging in will have no effect on this number? 
Alternatively could we rather track the number of current users that are logged instead?

Reply:
see above, this was placed here cos we don't have an api for creating users :/

re: tracking currently logged in users - yes, we can! good point and i'll add it in a future PR to avoid bloat here. (note that this metric is from product)